Components/History.js

```diff
@@ -1,17 +1,24 @@
 
 import React, { Component } from 'react';
-import { StyleSheet, Platform, View, Text, FlatList } from 'react-native';
+import { StyleSheet, Platform, View, Text, FlatList, TouchableOpacity } from 'react-native';
 import products from '../Helper/productData'
```

Comment:
@olivierag When you connect history to database, this is the line you should be able to remove.